cucumber/authenticators_jwt/features/authn_jwt_fetch_identity_decoded_token.feature

```diff
@@ -199,3 +203,76 @@ Feature: JWT Authenticator - Fetch identity from decoded token
     """
     cucumber:host:some_policy/host_test_from_token successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
+
+  @sanity
+  Scenario: ONYX-13707: Token-app-property from nested claim
+    Given I successfully set authn-jwt "token-app-property" variable to value "account/project/id"
+    And I am using file "identity-from-decoded-token" and alg "RS256" for remotely issue token:
+    """
+    {
+      "account":
+      {
+        "project":
+        {
+          "id": "myapp"
+        }
+      },
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with the JWT token
+    Then host "myapp" has been authorized by Conjur
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: ONYX-13711: Token-app-property does not accept array reference
+    Given I successfully set authn-jwt "token-app-property" variable to value "account[0]/project/id"
+    And I am using file "identity-from-decoded-token" and alg "RS256" for remotely issue token:
+    """
+    {
+      "account":
+      [
+        {
+          "project":
+          {
+            "id": "myapp"
+          }
+        }
+      ],
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    CONJ00117E Failed to parse 'token-app-property' value. Error: '#<Errors::Authentication::AuthnJwt::InvalidClaimPath: CONJ00116E
+    """
+
+  Scenario: ONYX-13713: Token-app-property does not accept array reference
```

Comment:
what will happened if the "token-app-property" variable is "accout"
Do we need a test for it ?

Reply:
This case is covered by UTs.
Actually I think that even this test as Integration is necessary but...